<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="filter_actions_label">
       <property name="text">
        <string>Filter action</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filter_actions_cb"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="filter_stacked_widget"/>
   </item>
   <item>
    <widget class="QPushButton" name="filter_btn_submit">
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
